src/indexer/utils/evm/index.ts

```diff
@@ -246,8 +248,20 @@ export async function saveDepositLogs(
   transferRepository: TransferRepository,
   depositRepository: DepositRepository,
   transferMap: Map<string, string>,
+  ofacComplianceService: OfacComplianceService,
+  accountRepository: AccountRepository,
 ): Promise<void> {
   let transfer = await transferRepository.findTransfer(decodedLog.depositNonce, Number(decodedLog.fromDomainId), Number(decodedLog.toDomainId))
+
+  const { sender } = decodedLog
+
+  const senderStatus = await ofacComplianceService.checkSanctionedAddress(sender)
+
+  await accountRepository.insertAccount({
```

Comment:
Will this error if the account already exists or insert a new account?

Reply:
I changed this to use `upsert` since it was giving an error